src/main/cpp/t04_students.cpp

```diff
@@ -29,10 +29,28 @@
 
 #include "t04_students.h"
 #include <iostream>
+#include <map>
+#include <vector>
 
 using namespace std;
 
 int t04_students() {
-    
+int N,z1,z2,z3;
+string x,y;
+float distance;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'distance' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4586019)